### DISCUSSION TITLE
Unable to connect intel RealSense camera Windows

### DISCUSSION THREAD

**Discussion by daomcgill** [PlusToolkit/PlusLib#965] [Q&A]
I am able to connect my Intel RealSense D455 if I use the Windows video capture device set, but not when using a device set file for Intel RealSense.
I had copied over the .xml from the examples in the Plus website, as shown in .txt form here: 
[PlusDeviceSet_Server_IntelRealSenseVideo.txt](https://github.com/PlusToolkit/PlusLib/files/9016508/PlusDeviceSet_Server_IntelRealSenseVideo.txt)
but am unable to figure out if I should have made changes to it, since Plus Server is says connection failed.

How do I fix this?

**Comment by Sunderlandkyl** [comment]
What version of Windows are you using? I don't think the RealSense software works on any OS before Windows 10.

Have you tried installing the librealsense SDK?
You can get it from the link on this page:
http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceIntelRealSense.html

**Reply by daomcgill** [reply]
I am running Windows 11 Home Pro, and have librealsense SDK installed. Should I be doing anything with the SDK after install? The camera works on the Intel Viewer.

**Reply by Sunderlandkyl** [reply]
I don't think we've tested the RealSense on Windows 11 yet.
Can you upload a log?

**Reply by daomcgill** [reply]
[070522_152433_PlusLog.txt](https://github.com/PlusToolkit/PlusLib/files/9050664/070522_152433_PlusLog.txt)
Here is the log. It says no device connected, but the device is definitely connected. Is it due to Win 11, or some other issue?
